src/CONST.js

```diff
@@ -1003,6 +1021,8 @@ const CONST = {
     CHAT_ATTACHMENT_TOKEN_KEY: 'X-Chat-Attachment-Token',
 
     USA_COUNTRY_NAME,
+    SPACE_LENGTH: 1,
+    SPACE: 1,
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/19289. In our world SPACE is still ` ` and not `1` :sweat_smile:.

Reply:
whoooops 😅

Reply:
😄